app/views/orders/past_orders.html.erb

```diff
@@ -0,0 +1,58 @@
+<%= render "header" %>
+<div id="full-order-page" class="full-order-page">
+  <div class="row orders present center">
+    <h4>Past Orders</h4>
+  </div>
+
+  <% if @past_orders.any? %>
```

Comment:
It's best practice to remove logic from our views when we can. You could try using a helper to do the logic and render the appropriate view